jsonrpclib/SimpleJSONRPCServer.py

```diff
@@ -164,12 +170,14 @@ def do_POST(self):
             L = []
             while size_remaining:
                 chunk_size = min(size_remaining, max_chunk_size)
-                L.append(self.rfile.read(chunk_size))
+                chunk = self.rfile.read(chunk_size).decode()
```

Comment:
Right. I'm not sure if there's any catch to using it. These chunks are always bytes from a socket? If they're encoded and decoded with the same encoding then does it all work out ok? I've assumed utf-8 at both ends for that I guess. The effect in python 2.7 seems to be; str was bytes anyway, so encoded to str, but decodes to unicode. Since unicode is also valid in the response there wasn't a problem there. Or am I missing something else/getting confused?